### ISSUE TITLE
Have searched concepts appear in hierarchy

### ISSUE THREAD

**Issue by Sigfried** [jhu-bids/TermHub#953]

**Comment by joeflack4** [comment]
@Sigfried this is done right?

**Comment by Sigfried** [comment]
No. What I meant by it is that when you're searching for concepts to add, you should see the matching concepts in a hierarchy -- at least vis-a-vis each other, if not also vis-a-vis the already selected concepts. But this was going to take some time and I never had a chance to implement it.
